File: Hero/Models/AutoAttachingEndorsementsRequest.cs
using Hero.Models.Enums;
using System.Collections.Generic;

namespace Hero.Models
{
    public class AutoAttachingEndorsementsRequest
    {
        public string ProductCode { get; set; } 
        public string CountryCode { get; set; }
        public string StateCode { get; set; }
        public int BrokerTeamId { get; set; }
        public int BrokerId { get; set; }
        public int BrokerGroupId { get; set; }
        public List<string> BusinessLineCodes { get; set; }
        public List<string> InsuringClauseCodes { get; set; }
        public List<string> InsuringClauseSectionCodes { get; set; }
        public List<string> ActivityCodes { get; set; }
        public bool HasSubjectivities { get; set; }
        public string LanguageCode { get; set; }
        public int WordingVersionId { get; set; }
        public InsuranceBasis BasisType { get; set; }
        public OriginSystem OriginSystem { get; set; }
        public Dictionary<string,string> RiskQuestionAnswers { get; set; }
        public string Coverholder { get; set; }
    }
}

File: Hero/ClientApp/src/app/quote/models/endorsements/AutoAttachingEndorsementsRequest.ts

import { InsuranceBasis, OriginSystem } from "@app/enums";

//no references to import
export class AutoAttachingEndorsementsRequest {
    public productCode: string;
    public countryCode: string;
    public stateCode: string;
    public brokerTeamId: number;
    public brokerId: number;
    public brokerGroupId: number;
    public businessLineCodes: string[];
    public insuringClauseCodes: string[];
    public insuringClauseSectionCodes: string[];
    public activityCodes: string[];
    public hasSubjectivities: boolean;
    public languageCode: string;
    public wordingVersionId: number;
    public riskQuestionAnswers: {};
    public documentBasisType: InsuranceBasis.Primary;
    public originSystem: OriginSystem;
    public coverHolder?: string;
}
